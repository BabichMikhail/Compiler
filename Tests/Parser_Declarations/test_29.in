const a = 10;
const b = 15;
const c = b - a;
var
	arr : array[c..a + b] of integer;