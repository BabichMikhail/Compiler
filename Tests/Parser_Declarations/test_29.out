Const	a
10

Const	b
15

Const	c
   a
-
   b

Var	arr
Array	
c
   b
+
   a
Type	integer

